### DISCUSSION TITLE
What am I?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#963] [Galaxy Trivia] [answered]
I speak without a 🙊 and hear without 🙉. I have no body, but I come alive with 💨.

**Comment by peterbaccaro** [comment]
The answer to the riddle is **an echo**.

An echo is a sound reflection that can be heard but does not possess physical form or a body. It is created when sound waves bounce off surfaces and return to the listener's ears. The reference to "coming alive with wind" suggests that echoes are more pronounced and noticeable in open spaces where wind can carry and reflect the sound waves effectively.

**Comment by CelioAmaral** [comment, marked as answer]
I speak, but I have no mouth. I hear, but I have no ears. I have no body, but I live with the wind. Who am I?

The answer to this riddle is the echo. For the echo "hears" what we speak, without having an ear of its own, and "speaks" back what we speak, without having a mouth. He does not have a body, and it is as if he lives like the wind.


When we say something and later manage to hear what we said again, it is because the sound emitted by us was reflected by obstacles, which characterizes the echo, and this occurs in 0.1s. We are able to perceive it because of the ability of the ears to distinguish which sound is initially made and which sound is reflected.
